src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/RichTextBoxTests.cs

```diff
@@ -10673,6 +10673,260 @@ public void RichTextBox_GetCharIndexFromPosition_Invoke_ReturnsExpected(int x, i
         Assert.Equal(0, createdCallCount);
     }
 
+    // DrawToBitmap doesn't work for this control, so we should hide it.  We'll
+    // still call base so that this has a chance to work if it can.
+    [WinFormsFact]
+    public void RichTextBox_DrawToBitmap_Invoke_Success()
+    {
+        using Bitmap bitmap1 = new(10, 10);
+        using RichTextBox richTextBox1 = new();
+        richTextBox1.DrawToBitmap(bitmap1, new Rectangle(0, 0, 10, 10));
+
+        bitmap1.Should().NotBeNull();
+        bitmap1.Width.Should().Be(10);
+        bitmap1.Height.Should().Be(10);
+    }
+
+    [WinFormsTheory]
+    [InlineData("SaveRichTextBox.rtf")]
+    public void RichTextBox_SaveFilePath_Invoke_Success(string fileName)
+    {
+        using RichTextBox richTextBox1 = new()
+        {
+            Rtf = @"{\rtf1\ansi{Sample for {\v HIDDEN }text}}"
+        };
+        string projectDirectory = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "..", "..", "..", "..", "..");
+        string filePath = Path.Combine(projectDirectory, "src", "System.Windows.Forms", "tests", "UnitTests", "TestResources", "Files", fileName);     
```

Comment:
You could write something like this;
```suggestion
        string filePath = $"{projectDirectory}/src/System.Windows.Forms/tests/UnitTests/TestResources/Files/{fileName}";     
```

https://learn.microsoft.com/en-us/dotnet/csharp/how-to/concatenate-multiple-strings#string-interpolation